lamindb_setup/dev/django.py

```diff
@@ -73,6 +75,12 @@ def get_migrations_to_sync():
     return status, latest_migrs
 
 
+def close_if_health_check_failed(self) -> None:
+    if self.close_at is not None and time.monotonic() >= self.close_at:
+        self.close()
+    self.close_at = time.monotonic() + CONN_MAX_AGE
```

Comment:
this should be only reset if it is not `None` i think.

Reply:
```
if self.close_at is not None:
  self.close_at = time.monotonic() + CONN_MAX_AGE
```

Reply:
Agreed! Fixed below!